alerter/src/main/java/org/apache/hertzbeat/alert/controller/AlertDefinesController.java

```diff
@@ -61,7 +64,13 @@ public ResponseEntity<Message<Page<AlertDefine>>> getAlertDefines(
             @Parameter(description = "List current page", example = "0") @RequestParam(defaultValue = "0") int pageIndex,
             @Parameter(description = "Number of list pages", example = "8") @RequestParam(defaultValue = "8") int pageSize) {
         Page<AlertDefine> alertDefinePage = alertDefineService.getAlertDefines(ids, search, priority, sort, order, pageIndex, pageSize);
-        return ResponseEntity.ok(Message.success(alertDefinePage));
+
+        return  ResponseEntity.ok(Message.success(
+                new PageImpl<>(
+                        alertDefinePage.getContent(),
+                        PageRequest.of(pageIndex, pageSize, Sort.by(new Sort.Order(Sort.Direction.fromString(order), sort))),
+                        alertDefinePage.getTotalElements()))
+        );
```

Comment:
Hi, it maybe caused by test cases or other reasons. I feel that we cannot repeat the construction like this.

Reply:
From the solution in the issue, this is one of the ways to solve the problem 🥲